<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bill</class>
 <widget class="QWidget" name="bill">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Счета</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Счета</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>261</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Del">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cho">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>550</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выбрать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
